cobigen-eclipse/cobigen-eclipse/.classpath

```diff
@@ -20,7 +20,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/mmm-util-core-7.4.0.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/slf4j-api-1.7.21.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/javaplugin-model-2.0.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/core-5.3.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/core-api-5.3.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/core-5.4.0-SNAPSHOT.jar"/>
```

Comment:
Why have you changed the version of the CobiGen core plugin? We should stick to `5.3.1` as 5.4.0 has not yet been released.